server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,229 @@
+            polling_status_code = polling_response.status_code
+            output_message(f"Polling GET WORKSPACE - {polling_status_code}")
+            if polling_status_code == 200:
+                output_message(f"Workspace {workspace_name} - {workspace_namespace} is still active")
+                retries_left -= 1
+            elif polling_status_code == 404:
+                is_workspace_deleted = True
+                output_message(f"Workspace {workspace_name} - {workspace_namespace} is deleted")
+            else:
+                output_message(f"Unexpected status code {polling_status_code} received\n{polling_response.text}")
+                raise Exception(polling_response.text)
+        if not is_workspace_deleted:
+            raise Exception(f"Workspace {workspace_name} was not deleted within {max_attempts * 2} minutes")
+        
+def test_cleanup(workspace_namespace, workspace_name):
+    try:
+        delete_workspace(workspace_namespace, workspace_name, 6)
+        output_message("Workspace cleanup complete")
+    # Catch the exeception and continue with the test since we don't want cleanup to affect the test results
+    # We can assume that Janitor will clean up the workspace if the test fails
+    except Exception as e:
+        output_message("Error cleaning up workspace, test script will continue")
+        output_message(f'Exception details below:\n{e}')
+
+def start():
+    workspace_namespace = ""
+    workspace_name = ""
+    workspace_id = ""
+    found_exception = False
+    
+    # Sleep timers for various steps in the test
+    workflow_run_sleep_timer = 60 * 5
+    provision_sleep_timer = 60 * 15
+    workflow_status_sleep_timer = 60 * 2
+    
+    try:
+        created_workspace = create_workspace()
+        workspace_id = created_workspace['workspaceId']
+        workspace_namespace = created_workspace['namespace']
+        workspace_name = created_workspace['name']
+        time.sleep(provision_sleep_timer) # Added an sleep here to give the workspace time to provision
+        app_url = get_app_url(workspace_id, 'cromwell')
+
+        # This chunk of code only executes one workflow
+        # Would like to modify this down the road to execute and store references for multiple workflows
+        workflow_response = submit_workflow_to_cromwell(app_url, "Run Workflow Test")
+        output_message(f'Executing sleep for {workflow_run_sleep_timer} seconds to allow workflow(s) to complete')
+        time.sleep(workflow_run_sleep_timer)
+
+        # This chunk of code supports checking one or more workflows
+        # Probably won't require too much modification if we want to run additional submission tests
+        workflow_ids = deque([workflow_response['id']])
+        get_completed_workflow(app_url, workflow_ids, workflow_status_sleep_timer)
+    except Exception as e:
+        output_message(f"Exception occured during test:\n{e}")
+        found_exception = e
```

Comment:
Should this be `True` rather than `e`? Looks like it's only ever being used as a boolean.